src/couch/src/couch_app.erl

```diff
@@ -14,8 +14,6 @@
 
 -behaviour(application).
 
--include_lib("couch/include/couch_db.hrl").
-
```

Comment:
Nice! I've been seeing various "header not used" warnings from erlang_ls in emacs but short of opening every module I couldn't figure out a way to performing a scan for the whole project at once.

Reply:
Same. I really didn't want to open every file individually.